django_prices_vatlayer/european_vat.py

```diff
@@ -0,0 +1,34 @@
+from decimal import Decimal
```

Comment:
Good question but being explicit about it for now doesn't hurt either I believe.